src/create.jl

```diff
@@ -1,4 +1,4 @@
-function create_connectivity(db::DB)::Connectivity
+function Connectivity(db::DB)::Connectivity
```

Comment:
Why aren't these snake case?

Reply:
They are constructors for the structs in solve.jl.